src/chart.zoom.js

```diff
@@ -41,32 +41,36 @@ function zoomIndexScale(scale, zoom, center) {
 
 }
 
-
 function zoomTimeScale(scale, zoom, center) {
 	var options = scale.options;
-	var minDelta, maxDelta;
-	var newDiff;
+	var range = scale.right - scale.left;
```

Comment:
OK.  I've added a draft implementation for Time scale 'Zoom to Cursor' on the Vertical.  Wasn't able to test though as I couldn't work out how to configure a Chart.js example with Time scale on the Vertical.